### ISSUE TITLE
AlignGraph getting stuck on Bowtie2-align

### ISSUE THREAD

**Issue by Anabaena** [baoe/AlignGraph#15]
So I've been running AlignGraph with 10 million paired reads, 160 Mb of scaffolds an a 350 Mb genome, but I've stumbled upon a problem.
For over a week now AlignGraph has not written ANYTHING. Not in the result or log files, nor in the tmp folder. When I look at the server that is running the program I only see some sleeping processes, among which "Bowtie2-align" which goes from 0 to 900.0% cpu every few seconds, but then back to 0 again.

Do you have any idea what may be wrong and how I could fix it? Because I'm quite sure this instance of AlignGraph won't finish.. Ever.

**Comment by baoe** [comment]
Maybe you could send me the command line for the AlignGraph run? Bowtie2-align should be extremely fast, so I guess the input might have some problem.

Best,
Bao

________________________________

**Comment by Anabaena** [comment]
Well, it did run for a bit, I do have quite a few files in my tmp folder, but then it just stopped doing anything..

This was my command:
AlignGraph --read1 Tt_WU10_L1_f1.fasta --read2 Tt_WU10_L1_r1.fasta --contig Aprime.fasta --genome TorRG33x02_asm01.scf.fasta --distanceLow 100 --distanceHigh 1100 --extendedContig test10M_extended.fa --remainingContig test10M_remaining.fa --misassemblyRemoval 1>log.test.out 2>log.test.err

**Comment by baoe** [comment]
I see. The bottleneck should be BLAT instead of Bowtie2. I'd suggest to add -fastMap option in the command line to accelerate BLAT and put PBLAT in your path so that AlignGraph can call PBLAT for parallel computation.

Best,
Bao

________________________________

**Comment by Anabaena** [comment]
I don't think so, from what I've seen it really was the Bowtie2 process that was failing.
I'm using PBLAT already, and I can't use the --FastMap option since my scaffolds are too long.

But I've cancelled my job and am rerunning it. Let's hope that it works now!

**Comment by baoe** [comment]
OK. It's a little strange that Bowtie2-align did not finish. Let me know if Bowtie2-align still could not finish.

Best,
Bao

________________________________
